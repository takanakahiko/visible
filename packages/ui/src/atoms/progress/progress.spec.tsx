import React from 'react';
import { render } from '@testing-library/react';
import { Progress } from '.';

describe('Progress', () => {
  it('matches snapshot', () => {
    const { container } = render(<Progress />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
