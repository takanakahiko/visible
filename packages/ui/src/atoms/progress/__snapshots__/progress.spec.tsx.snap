// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress matches snapshot 1`] = `
<div
  class="sc-bdVaJa EXKLM"
>
  <div
    class="sc-htpNat eCUBbA"
  />
  <div
    class="sc-bwzfXH iXiEnt"
    style="width: 50%;"
  />
</div>
`;
